setup.cfg

```diff
@@ -20,8 +20,9 @@ classifiers =
     Programming Language :: Python :: 3 :: Only
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Programming Language :: Python :: 3.11
+    # Support for Python versions 3.10 and above is temporarily disabled
+    # Programming Language :: Python :: 3.10 
```

Comment:
This should be highlighted as Jess/Jaclyn may work in 3.10 or 3.11.  Lets create a ticket to track potential upgrading to newer versions